pkg/astrolabe/data_transport.go

```diff
@@ -44,27 +52,109 @@ func NewDataTransport(transportType string, params map[string]string) DataTransp
 	}
 }
 
+const(
+	S3TransportType = "s3"
+	S3URLParam = "url"
+	S3HostParam = "host"
+	S3BucketParam = "bucket"
+	S3KeyParam = "key"
+	S3UseHTTPParam = "http"
```

Comment:
This constant was defined but not used. 

Maybe we can define one more constant, like, "https". Then, use them to replace hardcoded "http" and "https".